src/aat/java/uk/gov/hmcts/ccd/datastore/tests/AATHelper.java

```diff
@@ -13,37 +14,37 @@
     private final CCDHelper ccdHelper;
 
     AATHelper() {
-        idamHelper = new IdamHelper(getIdamURL());
+        idamHelper = new IdamHelper(getIdamURL(), OAuth2.INSTANCE);
         s2SHelper = new S2SHelper(getS2SURL(), getGatewayServiceSecret(), getGatewayServiceName());
         ccdHelper = new CCDHelper();
     }
 
     public String getTestUrl() {
-        return System.getenv("TEST_URL");
+        return Env.require("TEST_URL");
     }
 
     public String getIdamURL() {
-        return System.getenv("IDAM_URL");
+        return Env.require("IDAM_URL");
     }
 
     public String getS2SURL() {
-        return System.getenv("S2S_URL");
+        return Env.require("S2S_URL");
     }
 
     public String getCaseworkerAutoTestEmail() {
-        return System.getenv("CCD_CASEWORKER_AUTOTEST_EMAIL");
+        return Env.require("CCD_CASEWORKER_AUTOTEST_EMAIL");
     }
 
     public String getCaseworkerAutoTestPassword() {
-        return System.getenv("CCD_CASEWORKER_AUTOTEST_PASSWORD");
+        return Env.require("CCD_CASEWORKER_AUTOTEST_PASSWORD");
     }
 
     public String getGatewayServiceName() {
-        return System.getenv("CCD_GW_SERVICE_NAME");
+        return Env.require("CCD_GW_SERVICE_NAME");
     }
 
     public String getGatewayServiceSecret() {
-        return System.getenv("CCD_GW_SERVICE_SECRET");
+        return Env.require("CCD_GW_SERVICE_SECRET");
```

Comment:
How about we use apache commons Validate.notNull?

https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/Validate.html#notNull-T-java.lang.String-java.lang.Object...-

Reply:
Now using Validate.notNull